### DISCUSSION TITLE
Subscribe button -> Mail list

### DISCUSSION THREAD

**Discussion by Mat-moran** [shopinvader/odoo-shopinvader#1318] [Polls]
Hi, I would like to have an module that exposes an endpoint to subscribe to a mailing list? I did not see anything in shopinvader and maybe is an important feature for a website, what do you think?

**Comment by simahawk** [comment]
https://github.com/shopinvader/odoo-shopinvader/pull/1108
https://github.com/shopinvader/odoo-shopinvader/pull/1109

**Reply by Mat-moran** [reply]
thanks @simahawk  I did't see that. close this
